### PR TITLE
DOCS-8624-Scripting-index-edits-kt

### PR DESCRIPTION
**Pull request by kevintroller**
Scripting 2.0 Index doc

### CODE REVIEW

File: scripting/2.0/modules/ROOT/pages/edits/index.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+= Scripting Module 2.0 - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: connectors::scripting/scripting-module.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+The Scripting Module executes custom logic written in a scripting language. In some cases, you might need to create custom code to perform all or part of a complex task, or to reuse modules that you have already written.
```
Comment (kbostrom) [inline scripting/2.0/modules/ROOT/pages/edits/index.adoc]:
```suggestion
Scripting Module executes custom logic written in a scripting language. In some cases, you might need to create custom code to perform all or part of a complex task, or to reuse modules that you have already written.
```

File: scripting/2.0/modules/ROOT/pages/edits/index.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Since version 2.0 you must provide a compliant `jsr-223` scripting language engine.
```
Comment (kbostrom) [inline scripting/2.0/modules/ROOT/pages/edits/index.adoc]:
```suggestion
Starting with Scripting Module version 2.0, you must provide a compliant `jsr-223` scripting language engine.
```

File: scripting/2.0/modules/ROOT/pages/edits/index.adoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+For information about compatibility and fixed issues, see xref:release-notes::mule-runtime/module-scripting.adoc[Scripting Module Release Notes]
```
Comment (kbostrom) [inline scripting/2.0/modules/ROOT/pages/edits/index.adoc]:
```suggestion
For information about compatibility and fixed issues, see Scripting Module Release Notes.
```
Move link to See Also.

File: scripting/2.0/modules/ROOT/pages/edits/index.adoc [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+== Prerequisites
+
+To use this module, you must be familiar with the following concepts:
+
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements in a Mule flow
+* Global elements
+* Understand how to create a Mule app using Design Center or Anypoint Studio
```
Comment (kbostrom) [inline scripting/2.0/modules/ROOT/pages/edits/index.adoc]:
```suggestion
* Creating a Mule app using Design Center or Anypoint Studio
```

File: scripting/2.0/modules/ROOT/pages/edits/index.adoc [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+Before creating an app, you must have access to Anypoint Platform.
+
+
+== Common Use Cases For the Module
```
Comment (kbostrom) [inline scripting/2.0/modules/ROOT/pages/edits/index.adoc]:
```suggestion
== Common Use Cases for the Module
```

File: scripting/2.0/modules/ROOT/pages/edits/index.adoc [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+The Scripting Module performs logic in a supported language, which enables you to access variables and the registry, log information via the Mule logger, read payloads, and so on.
```
Comment (kbostrom) [inline scripting/2.0/modules/ROOT/pages/edits/index.adoc]:
```suggestion
Scripting Module performs logic in a supported language, which enables you to access variables and the registry, log information via the Mule logger, read payloads, and so on.
```

File: scripting/2.0/modules/ROOT/pages/edits/index.adoc [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+The Scripting Module executes custom logic written in a scripting language. In some cases, you might need to create custom code to perform all or part of a complex task, or to reuse modules that you have already written.
+Since version 2.0 you must provide a compliant `jsr-223` scripting language engine.
+
+For information about compatibility and fixed issues, see xref:release-notes::mule-runtime/module-scripting.adoc[Scripting Module Release Notes]
+
+== Prerequisites
+
+To use this module, you must be familiar with the following concepts:
```
Comment (kbostrom) [inline scripting/2.0/modules/ROOT/pages/edits/index.adoc]:
```suggestion
To use this module, you must be familiar with:
```

File: scripting/2.0/modules/ROOT/pages/edits/index.adoc [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+* Understand how to create a Mule app using Design Center or Anypoint Studio
+
+Before creating an app, you must have access to Anypoint Platform.
+
+
+== Common Use Cases For the Module
+
+The Scripting Module performs logic in a supported language, which enables you to access variables and the registry, log information via the Mule logger, read payloads, and so on.
+
+For example use cases, see xref:scripting-module-examples.adoc[Examples].
```
Comment (kbostrom) [inline scripting/2.0/modules/ROOT/pages/edits/index.adoc]:
```suggestion
For example use cases, see Examples.
```
Move link to See Also.

**Review by kbostrom — DISMISSED** [review]
Edited

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**